packages/codemod/src/upgrade-legacy.ts

```diff
@@ -33,8 +33,15 @@ const isInternalBlitzMonorepoDevelopment = fs.existsSync(
 type Step = {name: string; action: (stepIndex: number) => Promise<void>}
 const upgradeLegacy = async () => {
   let isTypescript = fs.existsSync(path.resolve("tsconfig.json"))
-  let blitzConfigFile = `blitz.config.${isTypescript ? "ts" : "js"}`
-  let isLegacyBlitz = fs.existsSync(path.resolve(blitzConfigFile))
+  let existingBlitzConfigFiles = ["ts", "js"]
+    .map((e) => `blitz.config.${e}`)
+    .map((e) => path.resolve(e))
+    .filter(fs.existsSync)
+  let blitzConfigFile = existingBlitzConfigFiles.reduce((_prev, current) => current, "")
+  if (blitzConfigFile === "") {
+    throw new ExpectedError("Could not identify Legacy Blitz Config file")
```

Comment:
No problem. As far as I can see, there is no other error later in the file regarding finding the config file.

Reply:
@oltdaniel This is what I meant https://github.com/blitz-js/blitz/blob/be39286393159dce2279fdfeec3b8da69652afdb/packages/codemod/src/upgrade-legacy.ts#L1211

Since we'll now throw the `ExpectedError()` at the top of the `upgradeLegacy` function like you have it.

Reply:
All that's left is the actual log.error ^ @oltdaniel